public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -278,15 +278,16 @@ export class UnConnectedExploreToolbar extends PureComponent<Props> {
               </div>
             )}
 
-            <div className="explore-toolbar-content-item explore-icon-align">
-              <ResponsiveButton
-                splitted={splitted}
-                title="Clear All"
-                onClick={this.onClearAll}
-                iconClassName="fa fa-fw fa-trash icon-margin-right"
-                disabled={isLive}
-              />
-            </div>
+            {!isLive && (
```

Comment:
If user is in live-tailing state, `clear all queries` is not visible, as disabled state was not very clear.

Reply:
Can we add a test for this? There are none for this component right now so no need to add more but would be good to start somewhere.